straxen/plugins/pulse_processing.py

```diff
@@ -150,38 +144,24 @@ def infer_dtype(self):
             if 'records' in p:
                 dtype[p] = strax.record_dtype(self.record_length)
         dtype['veto_regions'] = strax.hit_dtype
-        dtype['pulse_counts'] = pulse_count_dtype(self.config['n_tpc_pmts'])
+        dtype['pulse_counts'] = pulse_count_dtype(self.n_tpc_pmts)
 
         return dtype
 
     def setup(self):
-        self.hev_enabled = (
-                (self.config['hev_gain_model'][0] != 'disabled')
-                and self.config['tail_veto_threshold'])
+        self.hev_enabled = self.hev_gain_model is not None and self.tail_veto_threshold
         if self.hev_enabled:
-            self.to_pe = straxen.get_correction_from_cmt(self.run_id,
-                                           self.config['hev_gain_model'])
-
-        # Check config of `hit_min_amplitude` and define hit thresholds
-        # if cmt config
-        if is_cmt_option(self.config['hit_min_amplitude']):
-            self.hit_thresholds = straxen.get_correction_from_cmt(self.run_id,
-                self.config['hit_min_amplitude'])
-        # if hitfinder_thresholds config
-        elif isinstance(self.config['hit_min_amplitude'], str):
-            self.hit_thresholds = straxen.hit_min_amplitude(
-                self.config['hit_min_amplitude'])
-        else: # int or array
-            self.hit_thresholds = self.config['hit_min_amplitude']
```

Comment:
Since we are getting rid of these lines, should we remove [this](https://github.com/XENONnT/straxen/blob/55cea4ced3498181aee450357079e6d6f5b7b942/straxen/hitfinder_thresholds.py#L11) one as well?

Reply:
I think i still use that code in the protocol. Since its a shared function and not hardcoded into a plugin i saw no reason to remove it